src/components/Executions/ExecutionFilters.tsx

```diff
@@ -11,20 +18,47 @@ import {
     SingleFilterState
 } from './filters/types';
 
-const useStyles = makeStyles((theme: Theme) => ({
-    container: {
-        alignItems: 'center',
-        display: 'flex',
-        flexDirection: 'row',
-        height: theme.spacing(7),
-        minHeight: theme.spacing(7),
-        paddingLeft: theme.spacing(1),
-        width: '100%'
-    },
-    filterButton: {
-        marginLeft: theme.spacing(1)
-    }
-}));
+const useStyles = makeStyles((theme: Theme) => {
+    const buttonInteractiveStyles = {
+        borderColor: theme.palette.text.primary,
+        color: theme.palette.text.primary
+    };
+    return {
+        container: {
+            alignItems: 'center',
+            display: 'flex',
+            flexDirection: 'row',
+            height: theme.spacing(7),
+            minHeight: theme.spacing(7),
+            paddingLeft: theme.spacing(1),
+            width: '100%'
+        },
+        filterButton: {
+            marginLeft: theme.spacing(1)
+        },
+        button: {
```

Comment:
If we _were_ using a button, these styles should all already exist as part of the theme. I don't think we would want to repeat all of them here. But as I said below, I think we want to use a checkbox.